docs/src/components/ComponentPage/WebComponentPage.js

```diff
@@ -51,14 +51,14 @@ const WebComponentPage = (props: Props) => {
       id: example.id,
       title: example.title,
       blurb: example.blurb,
-      content: (
+      content: example.examples ? (
```

Comment:
Needed this because some examples will only have a blurb but no actual code — for example web images page.

Reply:
I'm confused by this. [All the examples](https://backpack.github.io/components/image/?platform=web) have a rendered component

Reply:
Argh, yeah sorry. I meant the icon page! Form elements has this too.